frontends/ol-widgets/src/components/Widget-RichText.spec.tsx

```diff
@@ -2,6 +2,8 @@ import React from "react"
 import { render, screen } from "@testing-library/react"
 import Widget from "./Widget"
 import { makeRichTextWidget } from "../factories"
+import { html_beautify as htmlBeautify } from "js-beautify"
```

Comment:
Just curious why this file is named `.spec.tsx` instead of `.test.tsx`?

Reply:
Oops. Habit. (jest by default treats either as a test file, but yes, we should use one or the other, and we use test.)

(It's easy to disable treating `.spec` as test files, but I'd only want to do that if we also had some sort of linting error for it.)